### ISSUE TITLE
Why I think the Mancrowave is an obstacle and not a good addition to Medbay

### ISSUE THREAD

**Issue by 9600bauds** [d3athrow/vgstation13#7909]
Now that Hypothermia is fully in and I've had a full round as MD to try out the Mancrowave, I want to detail out very clearly and very firmly what my experience was with it.
In order to get my point across, I'm going to tell you my experience with Medbay as a whole.
A lot of this might seem like it has nothing to do with the Mancrowave, but you need to understand what THE RIDE is like in order to know how the Mancrowave fits in with it.
At the end, I'll use my round story to pick up this point about the Mancrowave again.

I latejoin as Chemist, station time 12:18.
When I arrive, the other chemist is braindead, there's an assistant in Chemistry, and the fridge is completely empty. I mention to the doctors this and tell them I'm working on meds, to which they tell me the cryomix was apparently already done. Sweet.
I start making medicine, stock the fridge well, then go and look around Medbay. There are a few doctors and nothing seems to be happening so I go get a multitool to hack the vending machines and get an electrolyzer to make some Rezadone.

My memory fails me when I try to recall at what point THE RIDE started.
Then there was an explosion, so a lot of people were coming in with broken bones and external bleeding. I treated one man for his bone fracture and gave him blood.
When I peeked into Genetics, there was already a cloning queue, and I saw a man taking care of it.
There was a naked man with nearly all his blood missing, just inexplicably lying around in Medbay.
At around the same time there was a radstorm, so all of Medbay was swarmed with people vomiting everywhere and spewing even more radiation, which I had to treat with arithrazine, ryetelyn and antitox.
![dreamseeker_2016-01-27_20-22-05](https://cloud.githubusercontent.com/assets/6526157/12634535/1e82055e-c559-11e5-867c-c8f93f32db4f.png)

I suffered some radiation damage myself, which I ended up not curing in the entire shift for reasons soon to be made evident.
In the middle of all that, I notice that there's nobody doing cloning anymore, and there was a sizable pile of people to be cloned. Security told us not to clone a certain guy but I couldn't even tell him apart in the corpses. At this point, I was just constantly going to Genetics, ejecting the clone, feeding them Rezadone (imagine how much worse this would have been had I not powergamed rezadone) and throwing them into Cryo.
When throwing them into Cryo, I would just look for a full health bar in the Cryo displays, clickdrag the person out, clickdrag the clone in, then GO because I was so fucking busy, especially since the Rezadone was speeding things up. To be quite frank I didn't even REMEMBER the Mancrowave.
There were so many corpses in Genetics, at least six or seven, I just didn't know which ones were cloned or not because whoever did cloning before me just dumped the cloned corpses back in the same tile as the corpses that kept coming in. I decided to just throw them all in the Morgue and use the morgue tray indicators to see if they required cloning or not.
Micromanaging the best I could, I also drag the bloodless guy into Surgery to read his blood type, and start giving him blood. Come this point, unbeknownst to me, the cryomix that I assumed was taken care of did not have neither ryetalyn nor alkysine. I was then busy also giving rye and alk to everyone complaining about their "autism" aka brain damage.
There was a false alarm for a virus, which turned out to be someone coughing for some unknowable reason.
Inbetween short breaks, I would quickly try to work my way through the Genetics backlog, cloning people as fast as I could, which included just leaving the people I ejected from cryo lying on the ground because they're patient 2 in the list of 3 patients I'm trying to treat at once, possibly not even including the dead man I was giving blood to in the backroom. Once I realized that the Mancrowave was, in fact, a thing, I made an effort to try to put the recently cloned into it... It was no use. The thing is way too clunky, just outside of arm's reach which made dragging people to it bothersome, and in some cases nearly impossible because of the incredible amount of people loitering Medbay, some demanding their autism cured, some looking for their corpses, some BEING corpses.
I had to switch to disarm intent just to not be pushed away from the person I was trying to dunk into the microwave. A man demanded his craneal fracture fixed. After the microwave was done, the person I tried it on wouldn't respond, and I had no time to see if they still had hypothermia or not.
I noticed I had been starving for a while and never noticed before. At some point I made the terrible decision to fix someone's cranium and blood level rather than clone them, which only results in a bigger backlog of corpses.
My second attempt to put someone in the Mancrowave was stopped by a guy in an atmos hardsuit on a wheelchair blocking my way who wouldn't move, and upon trying to go around them I accidentally let go of my patient. After dragging them again I couldn't even sokoban not-help-intent push the mass of people out of my way. I quickly abandoned the idea of the mancrowave, and instead just hoped the recently cloned would be fine on the floor. Rail Kinson, who I didn't even know was on the station because I hadn't seen them around Medbay and especially not treating anyone, announces how they -think- they've made a cleaner foam grenade.
![dreamseeker_2016-01-27_20-23-05](https://cloud.githubusercontent.com/assets/6526157/12634529/1571f0f0-c559-11e5-8232-c554dfc9b45e.png)

I had to AFK for a couple of minutes. When I came back, I looked around again. Once again a sizable corpse pile had formed in Genetics, and the morgue was full. Someone over the radio said the borgs were rogue, and they looked rogue but I couldn't do anything about it.
I couldn't find a cryo cell available for one of the recently cloned, as both were taken by people with severe burn and brute damage, respectively (when did these people even come into medbay? Who put them in cryo fully clothed?). I ejected them, gave them some dermaline and bicaridine from the sleepers because I didn't know the state of the fridge, quickly confirmed the man with brute damage didn't have internal bleeding, and hoped they would be picked up by another doctor or self-medicate. I noticed that our front doors had been fucking knocked out somehow.
![dreamseeker_2016-01-27_20-34-06](https://cloud.githubusercontent.com/assets/6526157/12634540/25f6565a-c559-11e5-8bcb-e8b5342e89c1.png)

Someone over the radio was saying the CMO was a vampire. Once more I glanced over the idea of using the microwave and abandoned it, because despite my incredible hurry there were still people waiting to be cloned. I swear I've cloned two of these names not even ten minutes ago. We're almost out of Biomass, which is just spectacular because apparently an entire 30u bottle of Clonex was used for synthmeat. When I try to make Clonexadone I realize that the plasma is gone from Chemistry and I have to make it from dexaline from the emergency oxyloss kits. After what seems like a fucking odyssey I manage to make clonex and get the biomass back up. The power goes out from a random event. I spend a while trying to scrounge for a crowbar to open the door to Genetics (at this time I realize my entire backpack was completely empty, as well as my belt). I eject another man and while dragging him to cryo I am harmbatoned to death by two borgs at once because I was wearing shoes.
![dreamseeker_2016-01-27_20-44-14](https://cloud.githubusercontent.com/assets/6526157/12634544/348e61c6-c559-11e5-94dc-d5c228ad4566.png)

After an extraordinarily long wait and two shuttle recalls, I was finally cloned and put into Cryo. For some reason I took a while to fall asleep and could comment to the guy in literal knight's armor that the ride was extra wild today, while inside the cryotube, to which he agreed. After cryo, I was put in the mancrowave and after the five seconds passed I immediately sprung out in full health, with burn scars all over my chest. The "spring to life" characteristic of the mancrowave is even more jarring as a patient. For a while afterwards, I was walking at half speed, I assume some effect from hypothermia somehow? It eventually went away on it's own after I realized all the alkysine had ran out and I had misplaced my PDA in the cloning process so I couldn't make more. After I get my PDA back and make alkysine and more ryetalyn a man bitches at me because the cryomix I didn't do is making people retarded. I'm trying to sort out the situation, I clone two more people, and the shuttle finally arrives. The shuttle was plasma.

Now that you know what my day in Medbay was like, I want to revisit something that Celt has said a lot about his Mancrowave. According to @DrCelt, the Mancrowave actually helps in the Cryo process and speeds it up, because putting people in the Mancrowave is faster than shaking them back up. To this I've responded a few times that shaking people up is not part of the Cryo treatment, but maybe, just maybe you needed some context to the claim.
If you've read the wall of text (with pictures!) above, you will notice that I was very fucking busy and there was simply no physical time to handle cloning so many people at once. At NO point of the excercise, repeat at NO point did I stop to shake a single person up. The pacing of the ride demands that they be left alone and released as soon as they were fine, because there's Jesus Christ so many more people to treat.
Maybe Celt genuinely means well and he thinks the Mancrowave is a good addition to Medbay. I think that maybe he envisioned it in a hypothetical cloning scenario where one stray dude is brought into Medbay and cloned peacefully with no rush. Or maybe a well-staffed Medbay where everyone has a clear role and time to spare. Maybe the testing that went on just didn't reflect the reality of what goes on in Medbay in a daily basis.
Once again I have to remind you that, in a scenario where you're past your neck in corpses and you're the only one cloning them, you DON'T HAVE TIME for the clone after ejecting them (in my case, it was even worse since the cryomix wasn't healing them properly and I was too busy to fix it). In fact, the clone was half out of your mind the moment he was put in Cryo, you'll come to him later when you come back with a NEW clone. And much like the recently ejected didn't exist to me, neither did the Mancrowave, because neither did shaking them up exist before the Mancrowave was implemented. Had Hypothermia not been coded generously enough that these people didn't die on the floor, they would have probably died on the floor, and had I gotten used to remembering to microwave them every time, the entire cloning process would have been greatly slowed down.

In case all of this was not clear enough: The Mancrowave shaking people up for you does not simplify the cloning process. It does not make it shorter. It's an obstacle and it's sincerely one that I really want gone seeing as it has no real reason to exist, plusmore considering Medbay didn't need an obstacle anyways.

**Comment by ihadtoregisterforthis** [comment]
I can't believe I read the whole thing

**Comment by Kurfursten** [comment]
I don't think it's "powergaming" to make Rezadone from Groan's drinks. It's essentially your job as a chemist to go out of your way to get the good stuff - harass bartender for DD ingredients, botanists for plant stuff, hack vendors for pepto and rezadone, I always do it.

Railposting has officially entered github apparently, too.

**Comment by 9600bauds** [comment]
I think rezadone from trail mix zombie powder is a LIIIIIIIIIITTLE bit too easy honestly

**Comment by Kurfursten** [comment]
Oh, that's a neat strategy I didn't even think of. I figured you were doing Groan's Banned for Carpotoxin.

That being said, you're going far above the average chemist to strategically employ the electrolyzer at all.

**Comment by Shadowmech88** [comment]
If an alternative/faster way for reheating clones is desired, but without just making them magically pop out of cryo fully heated, the cryo tubes could be changed so that they have a slot for a beaker and a slot for a bottle.
If there's leporazine in the bottle, ejecting someone from the tube consumes a tiny bit of it and ejects the person with their body temp fully stable.

Now you've got a logical reason for why the person reheats and it takes a bit of prep to achieve, while also not adding to the time it takes to get them from the cloner out Medbay's doors.

**Comment by 9600bauds** [comment]
That's another thing that annoys me, the Mancrowave makes Leporazine useless to Medbay except for powergaming space travel/surviving plasmaflud. But I was going to open a more detailed issue on that (and all the ways the Mancrowave could be replaced) tomorrow.

**Comment by Shadowmech88** [comment]
Not _entirely_ useless, isn't it still useful for treating overheating?

**Comment by 9600bauds** [comment]
The microwave also cools you down
:V 

V:

**Comment by Shadowmech88** [comment]
> the microwave cools you down

I don't even know how to react to this.

**Comment by clusterfack** [comment]
It also wakes you up from sleeping

**Comment by Exxion** [comment]
It's a fucking thermal homeostasis regulator, not just for heating

Also, I'm not a medbay player so this doesn't hold much weight, but I've had to help out in medbay a couple times since it was added and I am a big fan of it. Far better than shaking people awake.

**Comment by Zth--** [comment]
I like the machine too. Today I had to treat some guys with 100+°C body temp and that thing solved my life

**Comment by 9600bauds** [comment]
For overheating, wouldn't a dunk in Cryo work too, anyways?

**Comment by Exxion** [comment]
Yes it would, but that's quite a bit more work than five seconds in the temperature box

**Comment by 9600bauds** [comment]
I'm not sure if I like that either to be honest
Cryo should definitely not need the 5 second temperature box
Fixing 100Cº+ bodytemp shouldn't be JUST 5 second temperature box, especially given that it's actually a rare condition all considered
Were these people affected by the "chili fucking murders you" bug?

**Comment by Zth--** [comment]
No. They were hacking the jukebox when they encountered the radiation wire. Yup, they got radiated and overheat.
It doesn't make sense though, anything above 50°C should be instakill

**Comment by 9600bauds** [comment]
>they got radiated and overheat.

????????????????

**Comment by Zth--** [comment]
Yup, I'll test that tomorrow. It didn't have sense to me when I saw it too

**Comment by 9600bauds** [comment]
I forgot to add the buttbot saying "butt IT THERE'S butt MANY CORPSES" which was the last thing I heard in Cryo

**Comment by gbasood** [comment]
tl;dr
(I'll actually read this tomorrow)

**Comment by ghost** [comment]
I don't think it's fair to blame the wild ride on the thermal homeostasis regulator. I've found, personally, it makes it easier. Perhaps you could consciously attempt to use it and see if it speeds it up? As for your other points, hmph, I'll have a think about them.

**Comment by ghost** [comment]
I believe overheating is caused by rapidly metabolizing nutriment, to the point of burning to death.

**Comment by 9600bauds** [comment]
Celt I really, really don't want to sound like a dick because even though I don't like the Mancrowave I have no doubt that you only meant good and wanted it to be a good addition, but you seem to have missed the point of what I said. I am not blaming the wild ride on the Mancrowave, I am saying that the Mancrowave is incompatible with the wild ride. Simply put, not using it is faster than using it.

**Comment by ihadtoregisterforthis** [comment]
The Mancrowave idea became mostly obsolete before the PR launch, when cryo was made not lethal if untreated.

**Comment by Exxion** [comment]
But how is something optional an obstacle? At worst, it's just unnecessary and you can choose whether or not to use it. And personally, I'd generally choose to.

**Comment by 9600bauds** [comment]
Celt himself said that he'd make Cryo hypothermia sometimes lethal if he could, and given his policy of "leave it underpowered at first then buff if later if needed" I am reasonably worried

**Comment by ghost** [comment]
@9600bauds if you want me to make a guarantee that cryo will not be made lethal in a balance change, you can have it.

**Comment by Iamgoofball** [comment]
i think this story reflects on how we really need competent medbay players

**Comment by PJB3005** [comment]
It also reflects on how we need a better way to throw nuisances OUT OF THE MEDBAY

**Comment by 9600bauds** [comment]
I DON'T HAVE TIME TO THROW THE NUISANCES OUT OF THE MEDBAY

**Comment by Iamgoofball** [comment]
@PJB3005 now that'd be nice if removing someone from medbay wasn't treated as valid escalation to murder you

**Comment by ghost** [comment]
Doctors need a cattleprod to handle the livestock, since they don't have a recharger it would run on blood.

**Comment by ghost** [comment]
Security officer stationed by default in medbay WHEN? I did this as a gimmick a few times and was quite successful.

**Comment by ghost** [comment]
Security has abysmal access rights.

**Comment by Zth--** [comment]
Yeah and they have tons of responsabilities too

**Comment by gbasood** [comment]
@DrCelt problem being that it's been very common recently to only have one sec officer half the rounds

**Comment by 9600bauds** [comment]
This is kind of not an issue anymore
